src/filters/regex_manager.rs

```diff
@@ -144,13 +144,7 @@ mod tests {
         regex_manager
             .get_debug_regex_data()
             .iter()
-            .fold(0, |acc, x| {
-                if x.regex == CompiledRegex::None.to_string() {
-                    acc
-                } else {
-                    acc + 1
-                }
-            })
+            .fold(0, |acc, x| if x.regex.is_some() { acc + 1 } else { acc })
```

Comment:
would be better expressed with a `filter` operation:

```rust
.filter(|x| x.regex.is_some())
.count()
```

Reply:
Fixed